docs/setup/docker-registry.mdx

```diff
@@ -26,6 +26,16 @@ docker pull registry.odigos.io/odigos-autoscaler:$VERSION
 docker pull registry.odigos.io/odigos-odiglet:$VERSION
 docker pull registry.odigos.io/odigos-collector:$VERSION
 ```
+In some cases, you will need to specify the processor architecture for the images.  For example, if you are deploying Odigos on Linux AMD64, but you are pulling the images to your laptop which uses ARM64, you will want to specify the platform in the `docker pull` commands:
```

Comment:
I would wrap this in a `<Warning></Warning>` tag [like so](https://github.com/odigos-io/odigos/blob/af294eef113b4cc0c5185bca7a2d43fa2e3b486b/docs/quickstart/introduction.mdx?plain=1#L28-L30) and rephrase it to say clearly `Odigos component images are published in multi-arch for ARM64 and AMD64 architectures. When pulling these images, be sure to specify the correct architecture *for the environment they will be deployed*.`